code-studio/src/js/assets/show.js

```diff
@@ -1,22 +1,23 @@
 /* global dashboard */
 
-var ImagePicker = require('./ImagePicker');
-var studioApp = require('../StudioApp').singleton;
+var ImagePicker = dashboard.ImagePicker;
```

Comment:
`ImagePicker` got moved to code-studio too, could do a `require` here instead of getting it from window.dashboard.